### PR TITLE
Leaderboard: Project setup branch

### PR DESCRIPTION
**Pull request by abenezer-addisu**
In This Branch :

- [ ]  we use Gitflow.
- [ ] we Create npm project with webpack. 
- [ ] we Write plain HTML markup with minimum styling

### CODE REVIEW

**Review by Zuhaib042 — COMMENTED** [review]
Hi @abenezer-addisu 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a readme file in the repo. kindly consider adding a readme file and fill all the sections with relevant information.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "leaderboard_01",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "watch": "webpack --watch",
+    "build": "webpack"
```
Comment (Ibizugbe) [inline package.json]: Currently, I think you don't have any script to run or start your server. hence, it is not possible to run your project on a server🥲. Kindly add the necessary script to make it possible to run your project on the server and ensure it is possible to run your project on a local server👍.

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @abenezer-addisu 👋👋, 

tryagain


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅✅

- [x] Linters are currently setup and they are passing👍.
- [x] Gitflow have been properly used👍.
- [x]  `node_modules` folder have been added to `.gitignore`👍.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by LionRouge1 — APPROVED** [review]
Hi @abenezer-addisu, 

# STATUS: APPROVED! 🥇 💯 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

done

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**